tests/embind/test_finalization.js

```diff
@@ -0,0 +1,7 @@
+Module.onRuntimeInitialized = () => {
+  const foo1 = new Module.Foo("Constructed from JS");
+  const foo2 = Module.foo();
+  const foo3 = Module.pFoo();
+}
+
+setTimeout(gc, 100);
```

Comment:
Ah, nice that this is so simple, node keeps the runtime alive for any setTimeouts automatically I see (tested locally too). I didn't know that.